requirements.txt

```diff
@@ -3,13 +3,13 @@ docopt == 0.6.2
 mediawiki-utilities==0.4.14
 mwparserfromhell == 0.3.3
 nltk == 3.0.1
-nose == 1.3.4
-numpy == 1.9.2
+nose >= 1.3.4
+numpy >= 1.9.2
 pytz == 2015.4
 requests >= 2.0.0, < 3.0.0
-setuptools == 15.1
+setuptools >= 15.1
 pyenchant == 1.6.6
-scipy == 0.14.0
-scikit-learn == 0.15.2
+scipy >= 0.14.0
+scikit-learn >= 0.15.2
```

Comment:
Well, how do you feel about scipy 0.15.1 and scikit-learn 0.16.1?

I'm using miniconda because the pip download was timing out.  Apparently that's a known problem.  Here's the list of package versions available through miniconda:
http://docs.continuum.io/anaconda/pkg-docs

IMO the best solution would be to port our app to work with the newer versions of these two modules, then lock == to those versions.